pom.xml

```diff
@@ -746,25 +746,6 @@
               </execution>
             </executions>
           </plugin>
-          <plugin>
-            <groupId>org.codehaus.mojo</groupId>
-            <artifactId>exec-maven-plugin</artifactId>
-            <executions>
-              <execution>
-                <id>check-shaded-content</id>
-                <goals>
-                  <goal>exec</goal>
-                </goals>
-                <phase>verify</phase>
-                <configuration>
-                  <executable>${basedir}/ci/scripts/check_content.sh</executable>
-                  <arguments>
-                    <argument>-thin</argument>

```

Comment:
note that here we are passing the additionall parameter that you miss in the profile

Reply:
Added new profile to run in the case of thin jar.